engine/runtime-compiler/src/main/java/module-info.java

```diff
@@ -0,0 +1,17 @@
+module org.enso.runtime.compiler {
+  requires java.logging;
+  requires scala.library;
+
+  requires org.enso.engine.common;
+  requires org.enso.pkg;
+  requires org.enso.runtime.parser;
+
+  requires org.openide.util.lookup.RELEASE180;
+  requires org.slf4j;
+
+  exports org.enso.compiler;
+  exports org.enso.compiler.context;
+  exports org.enso.compiler.data;
+  exports org.enso.compiler.dump;
```

Comment:
Who needs `dump`? I don't want to expose `dump` from the compiler. Definitely not as a stable API as astable as `org.enso.compiler`. Maybe just `export to`.